### PR TITLE
[benchmarks] Removed `benchmarks` module

### PR DESCRIPTION
**Pull request by DavidBakerEffendi**
Benchmarks have been moved to a standalone tool at https://github.com/joernio/joern-benchmarks

This is in preparation for a tool track paper submission.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -125,19 +125,3 @@ sbt stage
 ```
 The last command prints all available queries - add your own in querydb, run the above commands again to see that your query got deployed.
 More details in the [separate querydb readme](querydb/README.md)
```
Comment (mpollmeier) [inline README.md:127]: ## Benchmarks

Benchmarks have been moved to https://github.com/joernio/joern-benchmarks

**Review by mpollmeier — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (mpollmeier) [reply README.md:127]: I can't access that link btw - is it private?

Comment (DavidBakerEffendi) [reply README.md:127]: Yes it will be public shortly

Comment (DavidBakerEffendi) [reply README.md:127]: Seems I don't have permissions to change visibility, have pinged fabs

**Pull request merged**